supertokens_python/recipe/session/session_functions.py

```diff
@@ -59,6 +59,7 @@ async def create_new_session(
             "userId": user_id,
             "userDataInJWT": access_token_payload,
             "userDataInDatabase": session_data_in_database,
+            "useDynamicSigningKey": recipe_implementation.config.use_dynamic_access_token_signing_key,
             "enableAntiCsrf": enable_anti_csrf,
         },
     )
```

Comment:
why does `create_new_session` return `Dict[str, Any]`? 

We need to make all our functions strongly typed.. This applies to refresh_session and to get_session

Reply:
It was always like that. But I agree, I'll update them.

Reply:
Done in the PR that just got merged.